Reset ci environment back to trusty.

bkorty opened the pull request:
Reset the ci environment back to trusty because RabbitMQ is not currently
available in xenial environments, and is required to run tests.

bkorty committed:
Reset ci environment back to trusty.

Reset the ci environment back to trusty because RabbitMQ is not currently
available in xenial environments, and is required to run tests.

andrewrabert commented:
Python 3.7 isn't available on Xenial. This deploy section of the Travis config should be changed to Python 3.6 if we're going back to trusty.

Better yet, RabbitMQ could be Dockerized. https://docs.travis-ci.com/user/docker/

bkorty committed:
Remove python 3.7 from CI.

Remove python 3.7 from the ci-build. Though the library works in python 3.7,
travis-ci currently only supports python 3.7 in xenial environments, but the
xenial environment does not yet support the rabbitmq service.
